crates/runtime/src/continuation.rs

```diff
@@ -387,7 +387,9 @@ pub mod baseline {
     #[inline(always)]
     pub fn resume(instance: &mut Instance, contref: &mut VMContRef) -> Result<u32, TrapReason> {
         // Trigger fuse
-        HAS_EVER_RUN_CONTINUATION.set(true);
+        if HAS_EVER_RUN_CONTINUATION.get() {
```

Comment:
Should be
```suggestion
        if !HAS_EVER_RUN_CONTINUATION.get() {
```
right?

Reply:
Yes, you seem to be looking at an outdated commit

Reply:
I think you fixed it in the same instance as I submitted the review.